cloudsdk/src/test/java/io/particle/android/sdk/cloud/ParticleCloudTest.kt

```diff
@@ -76,15 +75,15 @@ class ParticleCloudTest {
     }
 
     @Test
-    fun test_generateClaimCode() {
+    fun test_generateClaimCode_HappyPath() {
```

Comment:
It's minor, but defining happy path might be more informative as test suite grows. Like `test_generateClaimCode_codeReceivedSuccesfully`. This particular endpoint is clear with just "happy path" but some endpoints have more vague result and for those it might be too little (pricing info for example?). If you agree, i suggest we adopt explicit names early to avoid refactoring in the future :)

Reply:
Agreed, I just didn't want to leave it without any qualifier as to _what was being tested_.